index.bs

```diff
@@ -762,6 +762,12 @@ spec: secure-contexts; urlPrefix: https://www.w3.org/TR/secure-contexts/
         required boolean supported;
         required boolean smooth;
         required boolean powerEfficient;
+        WebRTCMediaCapabilitiesInfo webrtc;
+      };
+
+      dictionary WebRTCMediaCapabilitiesInfo {
+        RTCRtpCodecCapability audio;
+        RTCRtpCodecCapability video;
```

Comment:
No hits for this dictionary. Do we mean [RTCRtpCodec](https://w3c.github.io/webrtc-pc/#dom-rtcrtpcodec).
```suggestion
        RTCRtpCodec audio;
        RTCRtpCodec video;
```